### ISSUE TITLE
confused about the service generated by operator

### ISSUE THREAD

**Issue by schecterdamien** [jaegertracing/jaeger-operator#990]
hi.  
 		
I follow the steps of the official website document to deploy step by step。there are some problem, the deploy.yml look like this:
    
```
apiVersion: jaegertracing.io/v1
kind: Jaeger
metadata:
  name: agent-as-daemonset
spec:
  agent:
    strategy: DaemonSet
    options:
      log-level: debug

```

jaeger run in observability namespace and my app run in another namespace. I want to run agent as daemonset and open a port in host machine so all my app could cross namespace pass span to agent，but:
   
<img width="845" alt="截屏2020-03-2619 49 36" src="https://user-images.githubusercontent.com/22927604/77644375-db6a1d00-6f9b-11ea-8d36-1014d544a7fe.png">

I find all service is ClusterIP，what i expect is agent-as-daemonset-agent service run in NodePort type，so i am confused.

my purpose is to run the application and jaeger in two different namespaces and could send span to the agent. Is there any correct way to do this?

**Comment by r0kas** [comment]
I second to this Issue, that there should be Service Type configuration available. Now it's left for default value which is not suitable for every use case.

Following link shows where the type is not configured and left to be at it's default setting: ClusterIP
https://github.com/jaegertracing/jaeger-operator/blob/3a76357a27208b29544890ab573b523717b00d32/pkg/service/query.go#L41

**Comment by GregoryVds** [comment]
Agreed. I would also like to deploy the Jeager query service as NodePort.
It seems most Helm charts expose a `service.type` option to configure such things.

**Comment by Aneurysm9** [comment]
> Agreed. I would also like to deploy the Jeager query service as NodePort.

I'll second this.  I'm trying to expose the query service through an ALB ingress and that expects the service to be of type NodePort.

**Comment by jpkrohling** [comment]
Alright, I think there's enough people who care about this. The best approach, IMO, is to create a new service. Who'd be the first to send a PR with a proposal? 

Basically, this here has to be changed to return a list of services, which will include the current service + the new service:
https://github.com/jaegertracing/jaeger-operator/blob/1dce126cdc3aa0f1582d53a6d62fcb359c34586c/pkg/service/query.go#L12-L53

The collector's code can be used as a reference: 
https://github.com/jaegertracing/jaeger-operator/blob/1dce126cdc3aa0f1582d53a6d62fcb359c34586c/pkg/service/collector.go#L15-L21

**Comment by jpkrohling** [comment]
I believe this is addressed by #1132, and thus, I'm closing this here. If you think there's still something missing, feel free to reopen.
